DashWallet/Sources/UI/CrowdNode/Online/OnlineAccountConfirmationController.swift

```diff
@@ -85,6 +85,8 @@ final class OnlineAccountConfirmationController: UIViewController {
 
 extension OnlineAccountConfirmationController {
     private func configureHierarchy() {
+        view.backgroundColor = .dw_secondaryBackground()
```

Comment:
By design, we have a white background with gray address boxes here: https://www.figma.com/file/aghEG2PAqmXghOWi5BoHg5/Dash-Wallet%E3%83%BBApproved%E3%83%BBAndroid?node-id=2159%3A31468&t=FLesXM2Oin3xbgCB-0

<img src="https://user-images.githubusercontent.com/7733989/218633068-d7cc45a4-dd44-4c56-b66f-1ce7879915b3.jpeg" width="300" />

Reply:
Fixed: Use 'QuaternaryFillColor' color for blocks like address